components/brave_wallet_ui/stories/wallet-concept.tsx

```diff
@@ -295,7 +296,7 @@ export const _DesktopWalletConcept = (args: { onboarding: boolean, locked: boole
     alert('Will update Watchlist')
   }
 
-  const onSelectSwapAsset = (asset: AssetOptionType, toOrFrom: string) => {
+  const onSelectTransactAsset = (asset: AssetOptionType, toOrFrom: string) => {
```

Comment:
Let's define `'to' | 'from'` somewhere, because it's used in a bunch of places.

Reply:
Added a type definition called `ToOrFromTypes` in the `types.ts` file to be reused.